### PR TITLE
Fix loading of language override setting. Fixes #1489

### LINKED ISSUE #1489: [Bug] Unable to change language settings
**Describe the bug**
When I set "non-system" language, then quit GRASS and start again, it is still in the "system" language.

**To Reproduce**

1. Go to Settings
2. Choose the "non-system" language 
3. Click save
4. Quit GRASS
5. Start GRASS again, the GRASS is still in the "system" language

**Expected behavior**
It should switch to "non-system" language.

**Screenshots**
VirtualBox_Windows_10__ArcGIS_Pro__31_03_2021_16_19_36

**System description:**

- Operating System: Windows
- GRASS GIS version master

### PR DESCRIPTION
**Pull request by marisn**
Setting of language override was broken as GUI now stores settings in a different file. This PR restores back old functionality by reading the setting from the new settings file.

### CODE REVIEW

File: lib/init/grass.py [hunk 1]
```diff
@@ -1373,19 +1374,11 @@ def set_language(grass_config_dir):
...
-        # Even if there is no override, we still need to set locale.
-        pass
-    else:
-        for line in fd:
-            if re.search("^language", line):
-                line = line.rstrip(" %s" % os.linesep)
-                language = "".join(line.split(";")[-1:])
-                break
-        fd.close()
+    with open(os.path.join(grass_config_dir, "wx.json"), "r") as json_file:
```
Comment (wenzeslaus) [inline lib/init/grass.py:1377]: Will fail if if wx.json does not exist like at a first start of GRASS by a user. Perhaps an additional/larger try-except with whatever open raises is the way to go.

File: lib/init/grass.py [hunk 2]
```diff
@@ -1373,19 +1374,11 @@ def set_language(grass_config_dir):
...
+        try:
+            language = json.load(json_file)["language"]["locale"]["lc_all"]
+        except KeyError:
+            language = None
```
Comment (wenzeslaus) [inline lib/init/grass.py:1381]: +1, more clear than the previous code. I'm not sure if `language == "None"` is still relevant with the new format. I would hope it is not.

**Review by wenzeslaus — CHANGES_REQUESTED** [review]
Looks good as far as replacing the outdated code except for that error handling of the _open_  function. Just for the record, I'm not looking much further (the function is almost 200 lines long).

**Comment by marisn** [comment]
> Just for the record, I'm not looking much further (the function is almost 200 lines long).

Better don't ;-) There is a reason why the code is so long – for every line there is a system where another path to failure was found.

File: lib/init/grass.py (hunk 1 above)
Comment (marisn) [reply lib/init/grass.py]: Thanks for catching this!

File: lib/init/grass.py (hunk 2 above)
Comment (marisn) [reply lib/init/grass.py]: Lets hope so. I removed this old workaround.

**Review by wenzeslaus — APPROVED** [review]

**Comment by landam** [comment]
@marisn Feel free to backport. Thanks.

**Comment by marisn** [comment]
Thanks, @neteler

**Pull request merged**